docs/source/notes/ddp.rst

```diff
@@ -193,3 +200,22 @@ DistributedDataParallel
 .. image:: https://user-images.githubusercontent.com/16999635/72313120-4e7c1c80-3658-11ea-9c6d-44336b2daeac.png
     :alt: ddp_code.png
     :width: 400 px
+
+TorchDynamo DDPOptimizer
+------------------------
+
+DDP's performance advantage comes from overlapping allreduce collectives with computations during backwards.
+AotAutograd prevents this overlap when used with TorchDynamo for compiling a whole forward and whole backward graph,
```

Comment:
I would imagine the DDP audience may not know what AotAutograd is, I'd rather expanding on this a bit more

Maybe a picture would help make things clearer

Reply:
Do you think it's better to duplicate the picture/explanation here, or would a link out to @davidberard98's  blog suffice?  He explains it well and has pictures

Reply:
Link is fine